includes/Classifai/Admin/SavePostHandler.php

```diff
@@ -228,9 +228,12 @@ public function synthesize_speech( $post_id ) {
 			);
 		}
 
+		// (SSML) document used for text-to-speech (TTS) conversion
+		$ssml_tag = "<speak version='1.0' xml:lang='en-US'><voice xml:lang='en-US' xml:gender='%s' name='%s'>%s</voice></speak>";
+
 		// Create the request body to synthesize speech from text.
 		$request_body = sprintf(
-			"<speak version='1.0' xml:lang='en-US'><voice xml:lang='en-US' xml:gender='%s' name='%s'>%s</voice></speak>",
+			apply_filters( 'classifai_azure_text_to_speech_ssml_tag', $ssml_tag ),
```

Comment:
@dkotter Do you think we can keep this filter hook? It is not related to the pull request context.

Reply:
I wonder if this was added to allow people to filter the `xml:lang` part? If not, I think this can be removed. If we do keep it though, we'll want a filter docblock added here

Reply:
I think we can remove it for now, and We should do it in a separate pull request.